test/integration_test.sh

```diff
@@ -28,159 +23,298 @@
+  cd $lira_dir
+  if [ $lira_version == "latest_released" ]; then
+    printf "\nDetermining latest release tag\n"
+    lira_version=$(python $script_dir/get_latest_release.py --repo HumanCellAtlas/lira)
+  elif [ $lira_version == "latest_deployed" ]; then
+    printf "\nDetermining latest deployed version\n"
+    lira_version=$(python $script_dir/current_deployed_version.py \
+                    --component_name lira
                     --env $env \
-                    --pipeline_name pipeline_tools)
-  echo "pipeline_tools_version: $pipeline_tools_version"
-#fi
+                    --mint_deployment_dir $mint_deployment_dir)
+  fi
+  printf "\nChecking out $lira_version\n"
+  git checkout $lira_version
+  cd $work_dir
+elif [ $lira_mode == "local" ]; then
+  printf "\n\nUsing Lira in dir: $lira_version\n"
+  lira_dir=$lira_version
+fi
 
-# 3. Build or pull Lira image
+# 3. Get pipeline-tools version
+if [ $pipeline_tools_mode == "github" ]; then
+  if [ $pipeline_tools_version == "latest_released" ]; then
+    printf "\n\nDetermining latest released version of pipeline-tools\n"
+    pipeline_tools_version=$(python $script_dir/get_latest_release.py --repo HumanCellAtlas/pipeline-tools)
+  elif [ $pipeline_tools_version == "latest_deployed" ]; then
+    printf "\n\nDetermining latest deployed version of pipeline-tools\n"
+    pipeline_tools_version=$(python $script_dir/current_deployed_version.py \
+                      --mint_deployment_dir $mint_deployment_dir \
+                      --env $env \
+                      --component_name pipeline_tools)
+  fi
+  printf "\nConfiguring Lira to use adapter wdls from pipeline-tools GitHub repo, version: $pipeline_tools_version\n"
+  pipeline_tools_prefix="https://raw.githubusercontent.com/HumanCellAtlas/pipeline-tools/${pipeline_tools_version}"
+elif [ $pipeline_tools_mode == "local" ]; then
+  pipeline_tools_prefix="/pipeline-tools"
+  pipeline_tools_dir=$pipeline_tools_version
+  cd $pipeline_tools_dir
+  pipeline_tools_dir=$(pwd)
+  cd $work_dir
+  printf "\n\nConfiguring Lira to use adapter wdls in dir: $pipeline_tools_dir\n"
+fi
+
+# 4. Build or pull Lira image
 if [ $lira_mode == "image" ]; then
-  if [ $lira_version == "latest" ]; then
-    lira_image_version=$(python $script_dir/get_latest_release.py HumanCellAtlas/secondary-analysis)
+  if [ $lira_version == "latest_released" ]; then
+    printf "\n\nDetermining latest released version of Lira\n"
+    lira_image_version=$(python $script_dir/get_latest_release.py --repo HumanCellAtlas/lira)
+  elif [ $lira_version == "latest_deployed" ]; then
+    printf "\n\nDetermining latest deployed version of Lira\n"
+    lira_image_version=$(python $script_dir/current_deployed_version.py lira)
   else
     lira_image_version=$lira_version
   fi
-  docker pull humancellatlas/secondary-analysis:$lira_image_version
+  printf "\nNot pulling lira_image_version: $lira_image_version -- fix this\n"
+  #docker pull humancellatlas/lira:$lira_image_version
```

Comment:
lira's image is on GCR now, we could either push it to docker hub or wait for quay migration.

Reply:
Good point, I pushed it to docker hub for now, just to get this script working as soon as possible. We can move it to quay later.